### PR TITLE
Add Rinkeby testnet addresses

### PR DESCRIPTION
**Pull request by alfetopito**
- [x] Deploy USDT to Rinkby using the shared team account: https://rinkeby.etherscan.io/address/0xa9881e6459ca05d7d7c95374463928369cd7a90c#code
- [x] Using existing USDC address and Compound's faucet
- [x] Using existing TUSD address and faucet provided by TUSD
- [x] Using existing WETH address
- [x] Deploy PAX to Rinkby using the shared team account: https://rinkeby.etherscan.io/address/0xbd6a9921504fae42ead2024f43305a8ed3890f6f#code
- [ ] Deploy GUSD to Rinkby using the shared team account
- [x] Using existing DAI address and Compound's faucet

### CODE REVIEW

**Comment by fleupold** [comment]
Any reason we are not using the Compound Rinkeby Addresses (at least for DAI and USDC).
They have a faucet in their UI and this would allow external people to also easily get and trade some tokens on the testnet.

Or are we planning on creating a faucet ourselves?

**Comment by alfetopito** [comment]
@fleupold wans't aware of that. Will check with @anxolin

**Comment by fleupold** [comment]
Also, are these MintableERC20 versions? How are we supposed to get balance otherwise?

**Comment by alfetopito** [comment]
The deployed contract is the exact copy of Tether's contract on mainnet https://etherscan.io/address/0xdac17f958d2ee523a2206206994597c13d831ec7#code

The contract allows the owner to mint (`issue` function) new tokens, if that's what you mean.

The account used is one that is shared with the team filled with some Rinkeby ether https://rinkeby.etherscan.io/address/0xf85d1a0e1b71e72013db51139f285c6d5356b712

**Comment by anxolin** [comment]
If there's an easy to find official version in rinkeby, with a faucet, it's good if we use that.
Otherwise I think we should deploy ourselves a copy that we can mint ourselves.

**Comment by alfetopito** [comment]
##Note

Couldn't find so far any official token deployed to Rinkeby for USDT, PAX nor GUSD.
- Contacted Tether support via email to ask for it. 
- Created issue on PAX github repo asking for it https://github.com/paxosglobal/pax-contracts/issues/4

In case there's no response by tomorrow or they do not provide one contract, I'll follow the approach of deploying a contract controller by us.

Will keep this issue on hold until then.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr68--dexreact.review.gnosisdev.com

**Comment by alfetopito** [comment]
Tether confirmed they do not provide an official Rinkeby contract. We'll stick to the one already deployed.

File: test/contracts/FakeTether.sol [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+
+contract UpgradedStandardToken is StandardToken {
+    // those methods are called by the legacy contract
+    // and they must ensure msg.sender to be the contract address
+    function transferByLegacy(address from, address to, uint value) public;
+    function transferFromByLegacy(address sender, address from, address spender, uint value) public;
+    function approveByLegacy(address from, address spender, uint value) public;
+}
+
+contract FakeTetherToken is Pausable, StandardToken, BlackList {
```
Comment (anxolin) [inline test/contracts/FakeTether.sol]: It's not a big deal, but I would use the original contract if possible. 

If we need to modify it, I would just extend the contract and modify only the parts that add the minting capabilities to our test user.

This way it's clear what's the original contract code, and what's ours

Comment (alfetopito) [reply test/contracts/FakeTether.sol]: New contract deployed, identical to original https://rinkeby.etherscan.io/address/0xa9881e6459ca05d7d7c95374463928369cd7a90c#code

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr68--dexreact.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr68--dexreact.review.gnosisdev.com

**Comment by alfetopito** [comment]
For some reason GUSD was named Paxos Gemini and because of that I thought both were by the same entity, which is not the case.

Anyway, looked around and couldn't yet find the source code for GUSD. Not visible on etherscan nor anything on their website about that.

I asked them via email.

In case they don't provide one, I'll write a basic ERC20 contract.

**Comment by alfetopito** [comment]
All stable coins deployed.

One problem with `GUSD` though. I cannot mint new tokens with the deployed contract. Will merge this PR and create a new task only for dealing with that.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr68--dexreact.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr68--dexreact.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr68--dexreact.review.gnosisdev.com

**Pull request merged**